backend/lcfs/web/api/notification/services.py

```diff
@@ -106,29 +105,62 @@ async def delete_notification_message(self, notification_id: int):
         else:
             raise DataNotFoundException(f"Notification with ID {notification_id}.")
 
+    @service_handler
+    async def get_notification_type_id_by_key(self, key: str) -> int:
+        notification_type = await self.repo.get_notification_type_by_key(key)
+        if not notification_type:
+            raise ValueError(f"Invalid notification type key: {key}")
+        return notification_type
+
+    @service_handler
+    async def get_notification_channel_id_by_key(self, key: str) -> int:
```

Comment:
Since there are only two channels for now, can we use ChannelEnum casting for key field?

Reply:
Of course!